rotkehlchen/api/v1/schemas.py

```diff
@@ -527,7 +528,7 @@ def _make_query(
             event_subtypes=query_event_subtypes,
             exclude_subtypes=exclude_event_subtypes,
             assets=asset_list,
-            entry_types=[HistoryBaseEntryType.HISTORY_EVENT],
+            entry_types=IncludeExcludeFilterData(values=[HistoryBaseEntryType.HISTORY_EVENT], behaviour='IN'),  # noqa: E501
```

Comment:
Here and everywhere else, since you have put a default, just use it.

```suggestion
            entry_types=IncludeExcludeFilterData(values=[HistoryBaseEntryType.HISTORY_EVENT]),  # noqa: E501
```